### PR TITLE
Add gem ransack.

### PR DESCRIPTION
**Pull request by dvbtham**

### CODE REVIEW

**Comment by dvbtham** [comment]
ready

File: app/controllers/movies_controller.rb [hunk 1]
```diff
@@ -42,6 +41,23 @@ def watch
 
   private
 
+  def ransack_params
+    Movie.newest.ransack params[:q]
+  end
+
+  def ransack_result
+    if :is_feature.to_s == params[:sort]
+      @search.result.where is_feature: true
```
Comment (VuThiTranVan) [inline app/controllers/movies_controller.rb]: `where is_feature: true` scope hì`

**Comment by dvbtham** [comment]
ready

**Comment by VuThiTranVan** [comment]
merged

**Pull request merged**